packs/data/book-of-the-dead-bestiary.db/deathless-acolyte-of-urgathoa.json

```diff
@@ -1959,24 +1959,39 @@
                     "value": null
                 },
                 "description": {
-                    "value": "<p>@Template[type:emanation|distance:20]{20 feet} @UUID[Compendium.pf2e.bestiary-ability-glossary-srd.Aura]{Aura}</p>\n<p>Allies within the aura who have negative healing gain @UUID[Compendium.pf2e.bestiary-ability-glossary-srd.Fast Healing]{Fast Healing 2}. When a creature within the aura would receive positive healing, it must succeed at a @Check[type:flat|dc:5] or receive no healing from the effect.</p>"
+                    "value": "<p>Allies within the aura who have negative healing gain fast healing 2. When a creature within the aura would receive positive healing, it must succeed at a @Check[type:flat|dc:5] or receive no healing from the effect.</p>"
```

Comment:
Generally we've left the template and aura links in place when adding aura REs. Should removing them be the standard going forward?

Reply:
It's rather silly to include a template that serves no other purpose than to misdirect the user.

Reply:
We can definitely revisit now that the aura RE exists, but we should have the aura size in the text as it is in the book.